### PR TITLE
Srinter 756

### PR DESCRIPTION
**Pull request by ooneill547**
Styled desktop dropdown menu that opens on hover over menu item

### CODE REVIEW

File: config/sync/structure_sync.data.yml [hunk 1]
```diff
@@ -215,8 +231,8 @@ menus:
     title: 'Information for...'
     parent: null
     uri: 'entity:node/2'
-    link_title: null
-    description: null
+    link_title: ''
+    description: 'Our quick start guides to help find  the right information for you.'
```
Comment (omahm) [inline config/sync/structure_sync.data.yml:235]: Looks like a double space between 'find' and 'the'

**Review by omahm — COMMENTED** [review]
Looks good in general. Scope to clean things up but the deadline is looming on this.

**Review by johangant — COMMENTED** [review]
Broadly looks fine to me as someone external to the project who hasn't got a mockup to view against 👍

**Review by DuttonMa — APPROVED** [review]
All looks good to me. I like the use of partials to simplify the primary menu as far as possible.

File: config/sync/structure_sync.data.yml (hunk 1 above)
Comment (ooneill547) [reply config/sync/structure_sync.data.yml:235]: That's what happens when you copy and paste from Photoshop... I'll tidy it up in the next pull request

**Pull request merged**